packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -399,7 +396,13 @@ export function UserListTable() {
     onColumnFiltersChange: setColumnFilters,
     getCoreRowModel: getCoreRowModel(),
     // TODO(SEAN): We need to move filter state to the server so we can fetch more data when the filters change if theyre not in client cache
-    getFilteredRowModel: getFilteredRowModel(),
+    // FIXME: How do I keep the filters working but still including the optionsGroup
```

Comment:
The filters seem to be working even without it, I guess because we are filtering through listMembers call already. Is it good to be removed @sean-brydon

Reply:
I had to remove it because now I would be showing a row with Territory=India for a filter of Territory=APAC which doesn't work if we use `getFilteredRowModel`

Reply:
nvm, looks like it is removed in main already :D